naga/src/front/glsl/functions.rs

```diff
@@ -641,8 +638,7 @@ impl Frontend {
                         self.errors.push(Error {
                             kind: ErrorKind::SemanticError(
                                 format!(
-                                    "'{}': image needs {:?} access but only {:?} was provided",
-                                    name, overload_access, call_access
+                                    "'{name}': image needs {overload_access:?} access but only {call_access:?} was provided"
```

Comment:
I'm nervous about some of these formatting string changes, because they almost certainly exceed the line limit that `rustfmt` normally imposes. This breaks `rustfmt` formatting for at least the given hunk of code, and often for more. Maybe we should suppress these instead? I guess that can be follow-up, though.